### DISCUSSION TITLE
how to toggle edgy window?

### DISCUSSION THREAD

**Discussion by cStralpt** [LazyVim/LazyVim#909] [Q&A] [answered]
i want to move my edgy window from left to right and set keymaps C-b to toggle edgy window hide/unhide.

is there any config that i could add?

**Comment by hirotaka** [comment]
I prefer it on the right side too!

I put following config as `lua/plugins/edgy.lua`.

```lua
return {
  "folke/edgy.nvim",
  opts = {
    right = {
      {
        title = "Neo-Tree",
        ft = "neo-tree",
        filter = function(buf)
          return vim.b[buf].neo_tree_source == "filesystem"
        end,
        size = { height = 0.5 },
      },
      {
        title = "Neo-Tree Git",
        ft = "neo-tree",
        filter = function(buf)
          return vim.b[buf].neo_tree_source == "git_status"
        end,
        pinned = true,
        open = "Neotree position=right git_status",
      },
      {
        title = "Neo-Tree Buffers",
        ft = "neo-tree",
        filter = function(buf)
          return vim.b[buf].neo_tree_source == "buffers"
        end,
        pinned = true,
        open = "Neotree position=top buffers",
      },
      {
        ft = "Outline",
        pinned = true,
        open = "SymbolsOutline",
      },
      "neo-tree",
    },
    left = {},
  },
}
```

I’m happy with the default `<leader>e`, so I don’t know about keymap.

**Reply by cStralpt** [reply]
leader-e only toggling neo tree, but not when we open neo tree git and others, it doesn't toggle the whole edgy window, like in vs code we can press C-b

**Reply by folke** [reply, marked as answer]
Please update edgy and lazyvim.

I added a keymap `<leader>ue` to toggle sidebars. The sidebar will open and load its pinned views.

**Reply by cStralpt** [reply]
thank u for developing lazyvim.

**Comment by folke** [comment]
To change the keymap, read the edgy docs https://github.com/folke/edgy.nvim

IT's in the config section under `keys`
